src/minimal-live-connect.js

```diff
@@ -52,15 +54,23 @@ function _minimalInitialization (liveConnectConfig, externalStorageHandler, exte
  * @param {LiveConnectConfiguration} liveConnectConfig
  * @param {StorageHandler} externalStorageHandler
  * @param {CallHandler} externalCallHandler
+ * @param {EventBus} externalEventBus
  * @returns {MinimalLiveConnect}
  * @constructor
  */
-export function MinimalLiveConnect (liveConnectConfig, externalStorageHandler, externalCallHandler) {
+export function MinimalLiveConnect (liveConnectConfig, externalStorageHandler, externalCallHandler, externalEventBus) {
   console.log('Initializing LiveConnect')
   try {
-    window && (window.liQ = window.liQ || [])
     const configuration = (isObject(liveConnectConfig) && liveConnectConfig) || {}
-    return _minimalInitialization(configuration, externalStorageHandler, externalCallHandler)
+    configuration.globalVarName = configuration.globalVarName || 'liQ'
+    window && (window[configuration.globalVarName] = window[configuration.globalVarName] || [])
```

Comment:
can we do this with an if? I really dislike this assignment inside a boolean expression